README.md

```diff
@@ -34,6 +34,14 @@ type.
 **Define styles in a stylesheet**
 
 ```swift
+enum Style: String, StringConvertible {
```

Comment:
Swift has `StringLiteralConvertible` http://nshipster.com/swift-literal-convertible/ which tells that the conformance type can be init with a string. This make me think that `Style` can be init with a string, perhaps `StringType` in this case should decrease confusion 😬